perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -1303,6 +1303,14 @@ sub construct_bin_request_form {
             ? 'request_removal'
             : ( $above_shop ? 'letterbox_location' : 'request_reason' );
 
+        unshift @$removal_field_list, (
+            no_removal => {
+                type         => 'Checkbox',
+                label        => 'None',
+                option_label => 'I do not need any bins to be removed',
+            }
+        ) if $include_removal;
```

Comment:
Just noting this for myself - this is here rather than in _construct_bin_request_form_removal because you only want it when it's used after the request step, not when they've asked for removal directly.

Reply:
Yeah 👍